cl/users/templates/profile/alerts.html

```diff
@@ -114,7 +114,9 @@ <h2>Search Alerts for Opinions and Oral Arguments</h2>
     <div class="col-xs-12">
       <h2>RECAP Docket Alerts for PACER Cases</h2>
       {% if user.profile.unlimited_docket_alerts %}
-        <h3 class="gray caps">Your account has unlimited docket alerts</h3>
+        <h3 class="gray caps">🎉 Your account has unlimited docket alerts</h3>
+      {% elif user.profile.email_grants_unlimited_docket_alerts %}
+        <h3 class="gray caps">🎉 Your email address provides you unlimited docket alerts</h3>
       {% endif %}
```

Comment:
```suggestion
     {% elif user.profile.is_monthly_donor %}
        <h3 class="gray caps">🎉 Being a monthly donor provides you unlimited docket alerts</h3>
      {% endif %}
```
Considering that being a monthly donor is another condition to have unlimited docket alerts, might be a good idea to also show a message?